CHANGELOG.md

```diff
@@ -0,0 +1,20 @@
+### 0.1.1
```

Comment:
is Betanet v2 just internal release name? or do we want to include it here.

Reply:
I guess it should be only considered internal? We can always update this changelog post-factum.